### PR TITLE
Discovery UI Updates

### PR DESCRIPTION
**Pull request by chasefleming**
Addresses: https://github.com/onflow/fcl-discovery/issues/36
------

### Items in Addition to Issue Spec

- Didn't want to break the UI existing apps using Discovery, so the configurable header piece with icon and title is versioned so that the old header shows (with a small change to account for the moving of the close button). A developer needs to update FCL in order to see the changes to the new configuration
- I added a message box to let the developer know on Testnet when there is missing configuration. This will also help in the case where a developer upgrade the version of FCL and sees Discovery change.
- The button change improvements show for any version

### Screenshots

Older versions of FCL with Discovery will see this:

Screen Shot 2022-03-09 at 2 51 34 PM

When they upgrade they'll see this with a warning if any app config is missing:

Screen Shot 2022-03-09 at 2 52 15 PM

When it's fully set they see this:

Screen Shot 2022-03-09 at 2 52 40 PM

### CODE REVIEW

File: components/Discovery.js [hunk 1]
```diff
@@ -80,73 +29,45 @@ const fetcher = (url, opts) => {
   }).then(d => d.json())
 }
 
-export const Discovery = ({network, appVersion, extensions, walletInclude, handleCancel}) => {
+export const Discovery = ({network, appVersion, extensions, walletInclude}) => {
```
Comment (gregsantos) [inline components/Discovery.js:32]: Maybe we can grab extensions off of config on next release, once we merge the account-proof-v2 stuff

**Review by gregsantos — APPROVED** [review]
SO Perty

**Review by JeffreyDoyle — APPROVED** [review]
🚢🚢🚢

File: components/Discovery.js (hunk 1 above)
Comment (chasefleming) [reply components/Discovery.js:32]: @gregsantos Definitely can

**Pull request merged**